warehouse/.gitignore

```diff
@@ -1,5 +1,8 @@
 .user.yml
 
+Miniconda3-py39_4.10.3-Linux-x86_64.sh
```

Comment:
are these files going to be created for everyone locally? they seem pretty specific for a gitignore

Reply:
If you are building the image yes, adding to docs